app/models/spree/adyen/notification_processor.rb

```diff
@@ -143,6 +143,12 @@ def create_missing_payment
         payment
       end
 
+      def payment_method_for(order)
```

Comment:
we shouldn't need to pass the order to this method since the class is initialized with it